integrate.js

```diff
@@ -65,6 +65,9 @@ WebApp._onPageReady = function()
 {
     // Connect handler for signal ActionActivated
     Nuvola.actions.connect("ActionActivated", this);
+    document.querySelector(".centerblock-wrapper").setAttribute('style', 'width: 100%');
```

Comment:
`document.querySelector()` may return `null` and `.setAttribute` would throw an error in that case. Either check the return value of  `document.querySelector()` not to be `null` or add `try { ...} catch (e){ /* ignored */ }`.